prow/tide/tide_test.go

```diff
@@ -3040,6 +3042,41 @@ func TestPresubmitsByPull(t *testing.T) {
 				},
 			}}},
 		},
+		{
+			name: "no-always-run-require-manually-triggered-jobs",
```

Comment:
Excluding last two examples most tests have english-like descriptions like "doesn't run against branch" or "no matching presubmits (check cache eviction)", so I would change your proposition to match english-like style that was presented previously.